prow/gerrit/adapter/adapter.go

```diff
@@ -183,6 +184,17 @@ func makeCloneURI(instance, project string) (*url.URL, error) {
 	return u, nil
 }
 
+// listChangedFiles lists (in lexicographic order) the files changed as part of a Gerrit patchset
+func listChangedFiles(changeInfo client.ChangeInfo) []string {
+	changed := sets.NewString()
+	for _, revision := range changeInfo.Revisions {
```

Comment:
I think we only care changed files on current revision, instead of all revisions?

Reply:
The client is only fetching the current revision anyway https://github.com/kubernetes/test-infra/blob/1920bc04207c6f3f3fdb2fcd4a5c39ced698ca7d/prow/gerrit/client/client.go#L269

Reply:
maybe also check for `revision.Number == changeInfo.CurrentRevision`? or add a comment? (the client code can be changed to add more additional fields and will cause undesired behavior)

Reply:
Changed it so it only gets files from current revision.